checker/check_result.go

```diff
@@ -85,6 +85,45 @@ type CheckResult struct {
 	Reason   string        `json:"-"` // A sentence describing the check result (score, etc)
 }
 
```

Comment:
ignore this file. Will disappear from the PR once https://github.com/ossf/scorecard/pull/739 is merged.